util-scripts/clean_db.sh

```diff
@@ -4,6 +4,6 @@
 # TODO: support purescript API rebuilding?
 
 rm -rf ./run/*
-rm -rf db-abc/
+rm -rf wallet-db/
```

Comment:
I am not sure that people who actually use this script (I don't use) don't mind having this change.